src/resources/js/event-editor.js

```diff
@@ -3,21 +3,143 @@ var tribe_events_event_editor = tribe_events_event_editor || {};
 /**
  * Implements behaviours that are specific to the event editor.
  */
-jQuery( function( $ ) {
-	var obj = tribe_events_event_editor,
-		$sticky_in_month_view_checkbox = $( 'input[name="EventShowInCalendar"]' ),
-		$featured_event_checkbox = $( 'input[name="feature_event"]' );
+( function ( $, obj ) {
+	'use strict';
+
+	obj = obj || {};
+
+	/**
+	 *	Setup our selectors.
+	 *
+	 *	@since TBD
+	 */
+	obj.selectors = {
+		$sticky_in_month_view_checkbox: 'input[name="EventShowInCalendar"]',
+		$featured_event_checkbox: 'input[name="feature_event"]',
```

Comment:
```suggestion
		sticky_in_month_view_checkbox: 'input[name="EventShowInCalendar"]',
		featured_event_checkbox: 'input[name="feature_event"]',
```
These shouldn't have `$` - and should be adjusted below to match.